books/free-programming-books-subjects.md

```diff
@@ -23,6 +23,7 @@ Books that cover a specific programming language can be found in the  [BY PROGRA
 * [Game Development](#game-development)
 * [Graphical user interfaces](#graphical-user-interfaces)
 * [Graphics Programming](#graphics-programming)
+* [Hacking](#hacking)
```

Comment:
It's usually not helpful to add a category for a single resource. Try to find another category, or see if there are other books in our lists that would fit into your category.